typings/index.d.ts

```diff
@@ -138,6 +138,34 @@ declare module 'discord.js' {
     public toJSON(...props: { [key: string]: boolean | string }[]): object;
   }
 
+  interface RouteBuilderMethodOptions {
+    query?: any;
```

Comment:
could there be a more specific type than `any`, afaik this accepts `URLSSearchParams` or a `Record<string, string | string[]>`